css-view-transitions-2/Overview.bs

```diff
@@ -517,17 +517,25 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 					We explicitly clear it here since the old Document may be cached by the UA.
 
 			1. [=Queue a global task=] on the [=DOM manipulation task source=] given |newDocument|'s [=relevant global object=],
-				to perform the following step:
+				to perform the following steps:
 
-				1. Let |newDocument|'s [=active view transition=] be a new {{ViewTransition}} in |newDocument|'s [=relevant Realm=],
+				1. Let |inboundTransition| be a new {{ViewTransition}} in |newDocument|'s [=relevant Realm=],
 					whose [=ViewTransition/named elements=] is |outboundTransition|'s [=ViewTransition/named elements=],
 					[=ViewTransition/initial snapshot containing block size=] is |outboundTransition|'s [=ViewTransition/initial snapshot containing block size=],
 					and whose [=ViewTransition/is inbound cross-document transition=] is true.
 
-				1. [=Call the update callback=] for |newDocument|'s [=active view transition=].
+				1. Let |newDocument|'s [=active view transition=] be |inboundTransition|.
+
+				1. [=Call the update callback=] for |inboundTransition|.
```

Comment:
I know this is the same as above, but is this just to resolve the promises since there is no update callback specified above when we declare what `inboundTransition` is

Reply:
Yes